packages/gdu/build.sh

```diff
@@ -3,15 +3,17 @@ TERMUX_PKG_DESCRIPTION="Fast disk usage analyzer with console interface written
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="5.28.0"
+TERMUX_PKG_REVISION=1
```

Comment:
No need to revbump this package. The previous CI doesn't push packages to aptly.